app/src/main/res/values/strings.xml

```diff
@@ -1010,7 +1010,7 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_shop_gone_vacant_answer">It’s vacant</string>
     <string name="quest_shop_gone_replaced_answer">A(n)…</string>
     <string name="quest_shop_gone_replaced_answer_hint">Type kind of place here</string>
-    <string name="quest_shop_gone_replaced_answer_error">"Don’t know what this is. Consider leaving a note."</string>
+    <string name="quest_shop_gone_replaced_answer_error2">"Enter only the type of object, for example 'bakery', not the name.\nLeave note if that fails."</string>
```

Comment:
I just noticed the bit about what it was before. So I think all you need is

```suggestion
    <string name="quest_shop_gone_replaced_answer_error2">"Unknown shop type. Consider leaving a note."</string>
```
I'm not sure if this is different enough to warrant a new string, or just marking the translation as fuzzy in POEditor.

Reply:
I think one of the main ideas was to explicitly **emphasize** to the user that they should  **not** enter the shop **name** in that field, **and** to give helpful **example** of what they need to enter. 

IMHO reducing all that to `Unlisted shop type` kinda misses those points (by making the explanation only implicit again, and also losing the example)...

Reply:
Yes, main problem that I noticed is that people try to enter given name ("Jolly good time" / "Lidl" / "Żabka" - not shop type such as "convenience" or "supermarket").

Old feedback was not mentioning this problem specifically and hint in background of the text field was not sufficient.

Reply:
I am aware that's the main problem. Compare my suggestion to the original, before this PR:

```diff
- Don’t know what this is. Consider leaving a note
+ Unlisted shop type. Consider leaving a note.
```

I think they both fix the problem, which is making it obvious what the problem is. Mine actually gives more detail, because it hints that StreetComplete only recognizes a fixed list of shops types (it's possible that the problem is because the user entered something overly specific, like "Wellness bar and cafe", which isn't in the list. @matkoniecz's suggestion wouldn't make it clear why that is failing.

I agree that we could add a few more words to mine in order to make it clearer, but I think the content is better.

Reply:
I'm afraid users will miss it in just `Unlisted shop type`. 
Example is crucial in my opinion, as is explicit instruction not to enter "name"... 
Maybe something along the lines of 
`Unlisted shop type; please enter a type like "bakery" and not a shop name, or leave a note if unsure.`